FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1080,7 +1077,7 @@ extension ComposeViewController {
     private func searchEmail(with query: String) {
         Task {
             do {
-                let localEmails = try contactsService.searchLocalContacts(query: query)
+                let localEmails = try localContactsProvider.searchEmails(query: query)
```

Comment:
For another issue in the future - should evaluate naming of methods in LocalContactsProvider. There's a method called `searchRecipient` (sounds good) but this one is called `searchEmails` (sounds like it should probably be called `searchRecipients` ? ) - to review

Reply:
But this one actually returns list of recipient emails.

Reply:
That sounds ok then